src/R.Settings.Paths.props

```diff
@@ -0,0 +1,10 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <!-- Common Build Settings and Configurations -->
+    <PropertyGroup>
+        <RootDirectory Condition="'$(RootDirectory)' == ''">$(MSBuildThisFileDirectory)..\</RootDirectory>
```

Comment:
Since `AssemblyName` is critical, we can add validation target. Something like:
```cs
    <Target Name = "ValidateRSettingsPaths" >
        <Error Condition = " '$(AssemblyName)'=='' " Text = "AssemblyName property must be specified before R.Settings.Paths.props is referenced" />
    </Target>
    <PropertyGroup>
        <BuildDependsOn>
            ValidateRSettingsPaths;
            $(BuildDependsOn)
        </BuildDependsOn>
        <BaseIntermediateOutputPath>$(ObjDirectory)\$(AssemblyName)\</BaseIntermediateOutputPath>
```